### PR TITLE
Line Chart: update demo description

### PR DESCRIPTION
**Pull request by sergepilipchuk**

### CODE REVIEW

File: JSDemos/Demos/Charts/Line/description.md [hunk 1]
```diff
@@ -1 +1,33 @@
-Line chart series types help visualize data trends over specified intervals. In&nbsp;this example, you can switch between the _line_, _stacked line_ and _full-stacked line_ series types. _Stacked line_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked line_ series help compare the percentage value of&nbsp;multiple line series for each argument.
\ No newline at end of file
+Line chart series types help visualize data and display them as lines with points placed over specified intervals. In this demo, you can see how to initialize and configure line chart of the *line*, *stacked line* and *full-stacked line* series types. 
+
+*Stacked line* series allow you to visualize multiple series and compare the manner in which each series contributes to the total aggregate value for specific arguments. *Full-stacked line* series help compare the percentage value of multiple line series for each argument.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
Line charts help visualize data and display them as lines with points placed over specified intervals. In this demo, you can see how to initialize and configure Line, Stacked Line, and Full-Stacked Line charts. A Stacked Line chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Line chart helps compare the percentage value of multiple line series for each argument. Use the drop-down menu under the chart to switch between chart types.
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 2]
```diff
@@ -1 +1,33 @@
...
+
+## Bind to Data
+
+You can bind the Line chart to one of the following data sources:
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
You can bind line charts to one of the following data sources: 
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 3]
```diff
@@ -1 +1,33 @@
...
+* [Local Array](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/)
+* [JSON Data](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Read-Only_Data_in_JSON_Format/)
+* [OData Service](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/OData/)
+* [Custom Data Source](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/)
+
+In this demo, the chart is populated with data taken from a local JavaScript array.
+
+## Configure Series
+
+Series define the look of chart. In this demo, the component includes Line, Stacked line and Full-stacked line series types. To display them, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
A series type defines the look of your chart. To display a Line, Stacked Line, or Full-Stacked Line series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types. 
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 4]
```diff
@@ -1 +1,33 @@
...
+
+In this demo, the chart is populated with data taken from a local JavaScript array.
+
+## Configure Series
+
+Series define the look of chart. In this demo, the component includes Line, Stacked line and Full-stacked line series types. To display them, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types. 
+
+You also need to bind the series to data. Set the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties to data fields that contain arguments and values for your series. You can specify these properties in an object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array or include it in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object. In this demo, the **argumentField** and **type** properties are specified for all series in the **commonSeriesSettings** object.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
You also need to bind the series to data. Set the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties to data fields that contain arguments and values for your series. You can specify these properties in an object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array or include them in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object. In this demo, the **argumentField** and **type** properties are specified for all series in the **commonSeriesSettings** object.
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 5]
```diff
@@ -1 +1,33 @@
...
+
+## Configure Axes
+
+Line chart allows you to configure the horizontal and vertical axes. To specify settings individually, you can use the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/) and [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) objects. You can also specify common settings for all axes in a chart, use the [commonAxisSettings](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/) object. Axis-specific settings override common settings.
+
+## Enable Tooltips
+
+When you hover the mouse pointer over a series point, you can see a tooltip with information about the series point. It shows information stored in the point value. To display tooltips, you need to enable the [tooltip.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) object. 
+
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
When you hover the mouse pointer over a series point, you can see a tooltip with information about the series point. To display tooltips, you need to enable the [tooltip.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. 
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 6]
```diff
@@ -1 +1,33 @@
...
+
+## Export Chart to Image
+
+To allow a user to print the chart or export it to a PNG, JPEG, or SVG file, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to *true*. This setting adds a button that opens a drop-down menu with export and print commands. In this demo, you can find this button in the upper-right corner.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
To allow a user to print the chart or export it to a PNG, JPEG, or SVG file, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to **true**. This setting adds a button that opens a drop-down menu with export and print commands. In this demo, you can find this button in the upper-right corner.
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 7]
```diff
@@ -1 +1,33 @@
...
+
+## Configure Axes
+
+Line chart allows you to configure the horizontal and vertical axes. To specify settings individually, you can use the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/) and [valueAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/valueAxis/) objects. You can also specify common settings for all axes in a chart, use the [commonAxisSettings](https://js.devexpress.com/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonAxisSettings/) object. Axis-specific settings override common settings.
```
Comment (RomanTsukanov) [inline JSDemos/Demos/Charts/Line/description.md]: We don't say about any axis features here. I think we better remove this section.

File: JSDemos/Demos/Charts/Line/description.md [hunk 8]
```diff
@@ -1 +1,26 @@
-Line chart series types help visualize data trends over specified intervals. In&nbsp;this example, you can switch between the _line_, _stacked line_ and _full-stacked line_ series types. _Stacked line_ series help visualize multiple series and compare the manner in&nbsp;which each series contributes to&nbsp;the total aggregate value for specific arguments. _Full-stacked line_ series help compare the percentage value of&nbsp;multiple line series for each argument.
\ No newline at end of file
+Line charts help visualize data and display them as lines with points placed over specified intervals. In this demo, you can see how to initialize and configure Line, Stacked Line, and Full-Stacked Line charts. A Stacked Line chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Line chart helps compare the percentage value of multiple line series for each argument. Use the drop-down menu under the chart to switch between chart types.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
Line charts help visualize data and display them as lines with points placed over specified intervals. This demo shows how to initialize and configure Line, Stacked Line, and Full-Stacked Line charts. A Stacked Line chart visualizes multiple data series and allows you to compare the manner in which each series contributes to the total aggregate value for specific arguments. A Full-Stacked Line chart helps compare the percentage value of multiple line series for each argument. Use the drop-down menu under the chart to switch between chart types.
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 9]
```diff
@@ -1 +1,26 @@
...
+## Bind to Data
+
+You can bind line charts to one of the following data sources: 
+
+* [Local Array](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Local_Array/)
+* [JSON Data](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Read-Only_Data_in_JSON_Format/)
+* [OData Service](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/OData/)
+* [Custom Data Source](/Documentation/Guide/Data_Binding/Specify_a_Data_Source/Custom_Data_Sources/)
+
+In this demo, the chart is populated with data taken from a local JavaScript array.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
In this demo, the chart is populated with data from a local JavaScript array.
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 10]
```diff
@@ -1 +1,26 @@
...
+
+## Configure Series
+
+A series type defines the look of your chart. To display a Line, Stacked Line, or Full-Stacked Line series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
The series type defines the appearance of your chart. To display a Line, Stacked Line, or Full-Stacked Line series, set the [type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/#type) property to one of these types. 
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 11]
```diff
@@ -1 +1,26 @@
...
+
+You also need to bind the series to data. Set the [argumentField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#argumentField) and [valueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#valueField) properties to data fields that contain arguments and values for your series. You can specify these properties in an object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) array or include them in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object. In this demo, the **argumentField** and **type** properties are specified for all series in the **commonSeriesSettings** object.
+
+## Enable Tooltips
+
+When you hover the mouse pointer over a series point, you can see a tooltip with information about the series point. To display tooltips, you need to enable the [tooltip.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
When you hover the mouse pointer over a series point, it shows a tooltip with information about the series point. To display tooltips, you need to enable the [tooltip.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. 
```

File: JSDemos/Demos/Charts/Line/description.md [hunk 12]
```diff
@@ -1 +1,26 @@
...
+
+## Enable Tooltips
+
+When you hover the mouse pointer over a series point, you can see a tooltip with information about the series point. To display tooltips, you need to enable the [tooltip.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#enabled) property. 
+
+## Export Chart to Image
+
+To allow a user to print the chart or export it to a PNG, JPEG, or SVG file, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to **true**. This setting adds a button that opens a drop-down menu with export and print commands. In this demo, you can find this button in the upper-right corner.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/Charts/Line/description.md]:
```suggestion
To allow a user to print the chart or export it as a PNG, JPEG, or SVG file, set the [export.enabled](/Documentation/ApiReference/UI_Components/dxChart/Configuration/export/#enabled) property to **true**. This setting adds a button that opens a drop-down menu with export and print commands. In this demo, you can find this button in the upper-right corner.
```

**Review by dirk-pieterse — DISMISSED** [review]

**Review by vladaskorohodova — APPROVED** [review]

**Pull request merged**